Updating scale section (#506)
PrestaShop/docs: src/content/1.7/scale/optimizations.md

### 1) PHP

> PHP tuning is very important for your application’s performance, either you’re running PrestaShop or any other PHP software.

Reviewer: ```suggestion
PHP tuning is very important for your application’s performance, whether you’re running PrestaShop or any other PHP software.
```

```
[opcache]
opcache.enable=1
opcache.enable_cli=0
opcache.memory_consumption=256
opcache.interned_strings_buffer=32
opcache.max_accelerated_files=16229
opcache.max_wasted_percentage=10
opcache.revalidate_freq=10
opcache.fast_shutdown=1
opcache.enable_file_override=0
opcache.max_file_size=0
```

If you can manage it, here are few options you could configure:

```
opcache.validate_timestamps=0
opcache.revalidate_path=0
```

> Keep in mind if you deactivate `validate_timestamps` OPCache will never update your code unless you let him know explicitly (either through internal functions or by restarting the webserver).

Reviewer: ```suggestion
Keep in mind if you deactivate `validate_timestamps` OPCache will never update your code unless you let it know explicitly (either through internal functions or by restarting the web server).
```

### 5) MySQL/MariaDB Settings

Though we’re discussing it just now, MySQL tuning is just as important.

Our intention here is to optimize the instance’s throughput by adding caches.

> As for PHP, it allows the service to work as much as possible in memory and avoid disk accesses, hence reducing latency.

Reviewer: ```suggestion
As for PHP, it allows the service to work as much as possible in memory and avoid disk access, hence reducing latency.
```

#### Caching

> Those parameters allow better cache information for further reuse, first by enabling it, then by increasing its sizes. 

Reviewer: ```suggestion
These parameters allow better cache information for further reuse, first by enabling it, then by increasing its size. 
```

Again, the idea is to keep the query results in memory rather than looking them up to the (higher latency) hard drive.

> As always, those values should be adapted to your own environment, you probably won’t need a ``host_cache_size`` of 1000. 

Reviewer: ```suggestion
As always, these values should be adjusted to your own environment, you probably won’t need a ``host_cache_size`` of 1000. 
```

```
query_cache_limit = 128K 
query_cache_size = 32M
query_cache_type = ON
table_open_cache = 1000
thread_cache_size = 80
host_cache_size=1000
```

#### Buffering

Buffering is almost another word for caching. 

So we work here with the memory area that holds cached data for InnoDB tables, indexes, and other auxiliary buffers, etc..

> Again, those options should be adapted to your shop. 

Reviewer: ```suggestion
Again, these options should be adapted to your shop. 
```

> Setting up the ```innodb_buffer_pool_size``` to 1G maybe too much for your SQL instance, just make sure you have enough memory according to the value you configure.

Reviewer: ```suggestion
Setting up the `innodb_buffer_pool_size` to 1G may be too much for your SQL instance, just make sure you have enough memory according to the value you configure.
```

> The important thing is, if possible, to set ```innodb_buffer_pool_size``` to a greater value than your database size.

Reviewer: ```suggestion
The important thing is, if possible, to set `innodb_buffer_pool_size` to a greater value than your database size.
```

```

> read_buffer_size			= 2M 
> read_rnd_buffer_size			= 1M
> join_buffer_size			= 2M 
> sort_buffer_size 			= 2M
> innodb_buffer_pool_size 		= 1G
> 

Reviewer: ```suggestion
read_buffer_size			= 2M 
read_rnd_buffer_size		= 1M
join_buffer_size			= 2M 
sort_buffer_size 			= 2M
innodb_buffer_pool_size 	= 1G

```

Author: Strange, yours are not aligned anymore Oo
Am I missing anything ?

Reviewer: It looks well on my end 🤔 

![Screenshot 2020-04-22 at 16 48 55](https://user-images.githubusercontent.com/1009343/79996839-31f24a80-84b9-11ea-951c-00b4d25bf5ad.png)

Author: That's a funny one ^^

![Capture d’écran 2020-04-23 à 00 51 17](https://user-images.githubusercontent.com/16487473/80041373-997db980-84fc-11ea-8e98-3af52398f611.png)
